mitodl/open-discussions
# Remove WEBHOOK_OCW flag, get-ocw-data from celery beat

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #3444 

#### What's this PR do?
Code cleanup:
 - Remove checks for `WEBHOOK_OCW` feature flag and assume it's always true. This feature flag is currently set on production, RC, and CI.
 - Remove `ignore_flag` from `get_ocw_data`. This argument essentially just exited the task without doing anything if `ignore_flag` was `False` (the default). There are two references to `get_ocw_data`: a task in celery beat without any options (so `ignore_flag` is `False`, meaning nothing happens), and `backpopulate_ocw_data` management command which calls `get_ocw_data` with `ignore_flag=True`. I removed the celery beat entry since it did not do anything.

#### How should this be manually tested?
It may be easier to review and merge https://github.com/mitodl/open-discussions/pull/3450 first. Then run `./manage.py backpopulate_ocw_data --course 6-890-algorithmic --overwrite --skip-s3` and verify that the corresponding `LearningResourceRun` was updated via the `updated_on` field: `Course.objects.get(url__contains='6-890-algorithmic').runs.first().updated_on` should be close to the current time
